### PR TITLE
Fixes #3560: Merge profile list fragment

### LINKED ISSUE #3560: Merge profile_list_fragment.xml into single xml file
Currently there are 3 versions of `profile_list_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by ShivamMadlani**
## Explanation
  Fixes #3560 by merging files

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing
- 
Before           |  After
:-------------------------:|:-------------------------:
port |  port1
Before | After
land |  land1
port | port
land | land

### CODE REVIEW

**Comment by rt4914** [comment]
Closing this PR. Please checkout our [wiki](https://github.com/oppia/oppia-android/wiki) first. This will help you in setting up the project and once you sign the CLA and fill the survey form we can assign you issues to work on.

**Comment by MaskedCarrot** [comment]
Reopening these PRs as Shivam has signed the CLA earlier with an incorrect GitHub handle. Sean corrected his handle.

File: app/src/main/res/layout/profile_list_fragment.xml
Comment (rishidyno) [inline app/src/main/res/layout/profile_list_fragment.xml]: Should be `profile_list_recycler_view_padding_bottom` instead.

**Review by rishidyno — CHANGES_REQUESTED** [review]
Rest of the pr LGTM. Just address the suggested changes.

File: app/src/main/res/values-land/dimens.xml
Comment (rishidyno) [inline app/src/main/res/values-land/dimens.xml]: Make it Pascal case. Ditto for others. Ex- `Profile List Fragment`.

**Review by rishidyno — CHANGES_REQUESTED** [review]
Rest of the pr LGTM. Just address the suggested changes.

**Comment by ShivamMadlani** [comment]
@rishidyno Made changes as per your comments. PTAL at them. Thanks!

**Comment by rishidyno** [comment]
@ShivamMadlani please add tablet Screenshots.

**Comment by ShivamMadlani** [comment]
@rishidyno I have added the screenshots of tablet view. PTAL.

**Review by BenHenning — DISMISSED** [review]
Thanks @ShivamMadlani. Two things:
1. The PR title should include the issue that's being fixed (see the PR description 'explanation' section).
2. It seems that per #3560 it's expected that all three ``profile_list_fragment.xml`` files be merged into one (the portrait, landscape, and tablet versions), but it seems only the landscape one is being merged into portrait here. Shouldn't the PR also be removing the tablet version & making sure only one layout file is needed?

**Comment by ShivamMadlani** [comment]
@BenHenning The tablet and the smartphone versions have many other changes other than just dimensions. Both the versions are using different layouts. Also the constraints of both the files vary and I don't think these can be merged. If they can then please let me know how.

**Comment by BenHenning** [comment]
Not stale--I need to follow up on this.

**Comment by BenHenning** [comment]
> @BenHenning The tablet and the smartphone versions have many other changes other than just dimensions. Both the versions are using different layouts. Also the constraints of both the files vary and I don't think these can be merged. If they can then please let me know how.

@ShivamMadlani apologies for the super late delay in my follow-up--I've been really slammed with various time-sensitive obligations lately.

The only differences I see between the handset & tablet versions of the layout is one difference in constraints and the addition of a toolbar & shadow for the handset version. @rt4914 is it not possible to move the toolbar bits to a common shared layout that we can just replace with a simple ``FrameLayout`` in tablet mode? That might also unlock merging other layouts that have the same constraint.

Furthermore, note that I dismissed my review since I'll be unavailable to follow-up, and @rt4914 has codeowners again for these files (so my review shouldn't be needed, anyway).

File: app/src/main/res/values-land/dimens.xml
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml:470]: Correct spacing at start and end of text

`<!-- Profile List Fragment -->`

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:674]: Correct spacing at start and end of text

`<!-- Profile List Fragment -->`

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:675]: Rename to `profile_list_fragment_recycler_view_padding_bottom `

File: app/src/main/res/values-land/dimens.xml
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml:471]: Rename to `profile_list_fragment_recycler_view_padding_bottom `

Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml:471]: Rename to `profile_list_fragment_recycler_view_padding_bottom `

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ShivamMadlani Thanks. Just suggested minor changes

**Comment by rt4914** [comment]
Closing this as it got solved in #4354

**Pull request closed without merging**
